### PR TITLE
modal tweaks (layout, icons etc.)

### PR DESCRIPTION
**Pull request by vikas-cis**
Fixed layout issues in modal titles, added icons and more small fixes & tweaks.

### CODE REVIEW

File: src/assets/theme.scss [hunk 1]
```diff
@@ -443,3 +443,8 @@ input {
 .capitalize {
   text-transform: capitalize;
 }
+
+// ------ Modal text lable alignment ------ //
+.modal-text-align {
+  margin-top: 20px !important;
```
Comment (allienworks) [inline src/assets/theme.scss]: This doesn't belong into `theme.scss`! Keep this file clean. If you want to add something that is shared across modals, there's a `app/modals/shared/_modal-layout.scss` instead.

Comment (vikas-cis) [reply src/assets/theme.scss]: actually this class is using in two components so I thought we can make it in once place

Comment (vikas-cis) [reply src/assets/theme.scss]: I got it (y)

**Comment by allienworks** [comment]
@vikas-cis also please add some descriptive description next time (for both PRs and issues)

File: src/assets/theme.scss (hunk 1 above)
Comment (allienworks) [reply src/assets/theme.scss]: Yes, that's why `_modal-layout.scss` is included in all modals.

**Comment by vikas-cis** [comment]
@allienworks I have attached screenshot, yeah may be I miss to add some lines in it 👍

**Pull request merged**